nft/nfts.yaml

```diff
@@ -601,6 +601,49 @@ paths:
                       type: number
                       description: 'Floating point value from 0 to 1 representing the prevalence of this value for this trait type.'
       operationId: computeRarity
+  '/{apiKey}/searchContractMetadata':
+    get:
+      summary: searchContractMetadata
+      description: Search for a keyword across metadata of all ERC-721 and ERC-1155 smart contracts
+      tags: ['NFT API Endpoints']
+      parameters:
+        - $ref: '#/components/schemas/apiKey'
+        - $ref: '#/components/schemas/query'
+      x-readme:
+        samples-languages:
+          - javascript
+          - curl
+          - python
+          - go
+      responses:
+        '200':
+          description: 'Returns the list of NFT contracts where the metadata overlaps with the search string.'
```

Comment:
What does "metadata overlaps" mean exactly? Is it when the metadata of an NFT contract contains the keywords provided in the search query? Can we update the phrasing to be more specific?